frontend/src/main/scala/bloop/Project.scala

```diff
@@ -52,10 +52,21 @@ case class Project(name: String,
 }
 
 object Project {
+
+  /** The pattern used to find configuration files */
+  final val loadPattern: String = "glob:**.config"
```

Comment:
We should change the `.config` ending at some point and favor something more bloop specific.